index.html

```diff
@@ -26,11 +26,11 @@ <h1>Hey, We are Brian and Michael</h1>
       <h2>We are currently working on this Project</h2>
         <div class="project-grid">
           <a href="">
-            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="Official portrait. 2012" class="project-img"/>
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="" class="project-img"/>
             <p class="project-tile">The Story of President Barrack Obama</p>
           </a>
           <a href="">
-            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg/1200px-US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg" alt="US President Barack Obama taking his Oath of Office - 2009Jan20.jpg" class="project-img"/>
```

Comment:
Please ensure that each of your image tags has an alt attribute. Alt attributes are important, especially for visually impaired people who depend on screen readers  to access your webpage.